hash/zx81_cass.xml

```diff
@@ -8693,4 +8878,20 @@ TZX files for ZX81 are just the .P dumps with an additional header on top, so th
 		</part>
 	</software>
 
+
+<!-- Homebrews -->
+
+	<software name="tuttut">
+		<description>Tut-Tut</description>
+		<year>2019</year>
+		<publisher>zx81keyboardadventure.com</publisher>
+		<info name="developer" value="David A. Stephenson" />
+		<info name="usage" value="16K" />
+		<part name="cass1" interface="zx81_cass">
+			<dataarea name="cass" size="15746">
+				<rom name="tut-tut.p" size="15746" crc="3af8b76f" sha1="228b11560f2937d7dfed0edc4b120fada36335f8"/>
+			</dataarea>
+		</part>
+	</software>
```

Comment:
Just use the name of the site as they write it in the header and footer – “ZX81 Keyboard Adventure”.